warehouse/config.py

```diff
@@ -60,12 +61,44 @@ class RootFactory:
     __name__ = None
 
     __acl__ = [
+        # TODO: Once all other Permissions are represented in the ACE, remove these.
         (Allow, "group:admins", "admin"),
         (Allow, "group:admins", "admin_dashboard_access"),
         (Allow, "group:moderators", "moderator"),
         (Allow, "group:moderators", "admin_dashboard_access"),
         (Allow, "group:psf_staff", "psf_staff"),
         (Allow, "group:psf_staff", "admin_dashboard_access"),
+        # New-style Permissions
+        (
+            Allow,
+            "group:admins",
+            (
+                Permissions.AdminBannerRead,
+                Permissions.AdminBannerPreview,
+                Permissions.AdminBannerWrite,
+                Permissions.AdminSponsorsRead,
+            ),
+        ),
+        (
+            Allow,
+            "group:moderators",
+            (
+                Permissions.AdminBannerRead,
+                Permissions.AdminBannerPreview,
+                Permissions.AdminSponsorsRead,
+            ),
+        ),
+        (
+            Allow,
+            "group:psf_staff",
+            (
+                Permissions.AdminBannerRead,
+                Permissions.AdminBannerPreview,
+                Permissions.AdminBannerWrite,
+                Permissions.AdminSponsorsRead,
+                Permissions.AdminSponsorsWrite,
+            ),
+        ),
```

Comment:
There's a possibility that we overload ourselves with too many fine-grained permissions - I don't know how many is too many.